metasmoke.py

```diff
@@ -81,12 +98,21 @@ def send_status_ping(self):
                 json_response = response.json()
                 commit_response = json_response["commit_status"]
                 if commit_response["status"] == "success":
+                    autopull_message = "Message contains 'autopull', pulling..."
```

Comment:
This seems really confusing to read. `{0} succeeded!` is... not the easiest to parse. Perhaps leaving it as it was is best.